wikimedia/wikipedia-ios
# [ALT TEXT] Add "Set experiment to 100%" developer settings toggle

**Phabricator:** https://phabricator.wikimedia.org/T372098

### Notes
This PR adds a simple toggle and logic for setting the experiment percentage to 100% for easier testing. I also tried bypassing the modal prompt flags and "already assigned experiment" checks, but it was acting a bit strange. I didn't want to spend much time investigating a temporary testing workaround, so I'm just keeping this strictly to the experiment percentage for simplicity.

### Test Steps
1. On fresh install, turn on the new developer settings toggle.
2. Confirm you can go through an image recommendation and always see the alt text prompt.
3. Regression: Fresh install again, leave the new developer settings toggle off. Confirm when you go through an image recommendations edit, _sometimes_ (50%) you do not see the alt text prompt.
